core/templates/dev/head/services/assets-backend-api.service.ts

```diff
@@ -279,6 +290,10 @@ angular.module('oppia').factory('AssetsBackendApiService', [
       getImageUrlForPreview: function(entityType, entityId, filename) {
         return _getDownloadUrl(
           entityType, entityId, filename, ASSET_TYPE_IMAGE);
+      },
+      getTopicThumbnailUrlForPreview: function(entityType, entityId, filename) {
```

Comment:
This service should be unaware of "Topic" word, so I would suggest you use `getThumbnailUrlForPreview`.

Reply:
Done.